src/app/test-record/test-section/test-section.component.html

```diff
@@ -39,19 +39,29 @@
       {{ testType.additionalCommentsForAbandon | DefaultNullOrEmpty | CapitalizeString }}
     </dd>
   </div>
+  <ng-container *ngIf="testTypesApplicable">
+    <div *ngIf="hasCertificateNumber" class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        Certificate number
+      </dt>
+      <dd class="govuk-summary-list__value" id="test-certificateNumber">
+        {{ testType.certificateNumber | DefaultNullOrEmpty }}
+      </dd>
+    </div>
+  </ng-container>
   <ng-container *ngIf="testTypesApplicable">
     <div
       *ngIf="
-        testTypesApplicable.anniversaryDateApplicable[testType.testTypeId] ||
-        testTypesApplicable.certificateExtraApplicable[testType.testTypeId]
+        testTypesApplicable.specialistCOIFApplicable[testType.testTypeId] &&
```

Comment:
What about moving those into a service now?
They are very likely to be reused across and we could easily be testing them.
We could also be adding the future ones we will be having depending on the groups.

Reply:
reference ticket: https://jira.dvsacloud.uk/browse/CVSB-16312